storybookjs/storybook
# Addon-backgrounds: Customizable Grid Cell Size

Issue: #5853 #6252

## What I did

- The functionality is lost during the large refactoring that happened between v5.1 and v5.2
- Added a param for the Grid selector: `grid`, with a single option `{ cellSize: number }`
  - The original implementation in #6252 used the theme object, but from what I understand this should be converted to a param after refactoring into an addon (which is done), so I chose to do it this way instead.
  - I can also remove the param from the theme and remove the relevant tests, but it will be a breaking change. Should I also allow consumption of the theme variable?
- Switched the backgrounds to use `backgroundColor` instead of `background`, because the latter would override the grid css
- Fixed the `rgba` issue in the grid css

## How to test

- Is this testable with Jest or Chromatic screenshots? **Yes**
  - A story regarding this change is added to the official storybook
- Does this need a new example in the kitchen sink apps? **Probably no**
- Does this need an update to the documentation? **Unsure**
  - If needed, I can also document it in the README of the addon

If your answer is yes to any of these, please make sure to include it in your PR.
